### PR TITLE
Update changelog with more details for gRPC support

### PR DESCRIPTION
**Pull request by alanwest**
@cijothomas Additional details based on our conversation. Sorry, forgot to include in #803.

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 ## Unreleased
 
 * Introduced support for Grpc.AspNetCore (#803).
+  * Attributes are added to gRPC invocations: `rpc.system`, `rpc.service`,
+    `rpc.method`. These attributes are are added to an existing span generated
```
Comment (eddynaka) [inline src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md:7]: we have two "are"s

File: src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
...
+    by the instrumentation. This is unlike the instrumentation for client-side
+    gRPC calls where one span is created for the gRPC call and a separate span
+    is created for the underlying HTTP call in the event both gRPC and HTTP
+    instrumentation is enabled.
```
Comment (eddynaka) [inline src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md:11]: Maybe we should switch from is enabled to "are enabled". I'm not sure (english isn't my first language)...

File: src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md (hunk 1 above)
Comment (alanwest) [reply src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md:7]: Arrrr, matey! Good catch. 🏴‍☠️

File: src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md (hunk 2 above)
Comment (alanwest) [reply src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md:11]: Arrrr! Your English is perfect.

**Pull request merged**